packages/@react-aria/visually-hidden/src/VisuallyHidden.tsx

```diff
@@ -30,14 +30,16 @@ interface VisuallyHiddenProps extends HTMLAttributes<HTMLElement> {
 
 const styles: CSSProperties = {
   border: 0,
-  clip: 'rect(0 0 0 0)',
-  clipPath: 'inset(50%)',
-  height: 1,
-  margin: '0 -1px -1px 0',
+  clip: 'rect(0 10px 10px 0)',
+  clipPath: 'inset(40%)',
```

Comment:
Unfortunately due to this clipPath no longer being `inset(50%)` I am able to click in the top left corner of the datepicker modal and close it: https://reactspectrum.blob.core.windows.net/reactspectrum/868e245a81fbaea15da6756323a666be62a5460b/storybook/index.html?path=/story/date-and-time-datepicker--default

@majornista I think I remember you saying that this `clipPath` change was necessary to get your Android devices to properly click the visually dismiss button via TalkBack? If so we'll have to discuss if we think this is a big enough problem to ditch this approach. Maybe we could absolutely position the visually hidden dismiss button just outside of the Dialog, not sure...

Reply:
We may need to ditch the approach or at least modify it. I've oftentimes thought the dismiss button should just be a big hit target placed over the visible viewport dimensions, underneath the overlay.